lib/pages/chat/chat.dart

```diff
@@ -522,6 +523,10 @@ class ChatController extends State<Chat>
   }
 
   void onFileTappedMobile(Event event) async {
+    if (Platform.isAndroid &&
+        (await DeviceInfoPlugin().androidInfo).version.sdkInt >= 33) {
```

Comment:
i think you should use PlatformInfos and check sdkInt in permission handler

Reply:
fixed